peakina/readers/csv.py

```diff
@@ -0,0 +1,51 @@
+"""
+Module to add csv support
+"""
+from typing import Any, Optional, TypedDict
+
+import pandas as pd
+
+# The chunksize value for previews
+PREVIEW_CHUNK_SIZE = 1024
+
+
+class PreviewArgs(TypedDict, total=False):
+    nrows: int
+    offset: int
+
+
+def read_csv(
+    filepath: str,
+    *,
+    sep: str = ",",
+    keep_default_na: Any = None,
```

Comment:
```suggestion
    keep_default_na: bool = True,
```
https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html

Reply:
👍🏾  Updated !